c8/apikeys.py

```diff
@@ -58,9 +63,28 @@ def response_handler(resp):
             else:
                 return resp.body
                 
-        return self._execute(request, response_handler)
+        return self._execute(request, response_handler, customPrefix="/_api")
```

Comment:
It is not a huge error or even an error but is it possible to name customPrefix in a more pythonic(PEP8) way such as custom_prefix? If it is an issue we can perhaps do it later

Reply:
Sure I will do this in a separate PR tomorrow.

Reply:
👍